deb/openmediavault/workbench/src/app/core/components/intuition/abstract-page-component.ts

```diff
@@ -49,10 +44,7 @@ export abstract class AbstractPageComponent<T> implements AfterViewInit, OnInit,
   @Output()
   readonly afterViewInitEvent = new EventEmitter();
 
-  readonly routeUrlSegments: string[];
-  readonly routeConfig: Route;
-  routeParams: Params = {};
-  routeQueryParams: Params = {};
+  public pageContext: PageContext = inject(PageContextService).getContext();
```

Comment:
Please use old-style dependency injection via constructor. All code in this project is doing it this way, so there is no reason to mix old and new style. If new style is preferred, all code needs to be refactored.